.circleci/config.yml

```diff
@@ -105,7 +105,7 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - mypy-{{ .Branch }}-{{ checksum "setup.cfg" }}-{{ checksum "/home/circleci/.pyenv/version" }}
+            - mypy-{{ checksum "setup.cfg" }}-{{ checksum "/home/circleci/.pyenv/version" }}
```

Comment:
In favor of the approach in general, but currently, we also catch new issues resulting from new versions of deps when we create new PR

Reply:
okay, I mean if we add a new dep to setup.cfg explicitly that would change the checksum for setup.cfg. But, I get your second point where since we don't have pinned dependencies, we might not update the deps to new versions and to try that out makes sense we create a new branch and avoid using cache.